docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -45,11 +45,11 @@ Follow the steps below by clicking on the togglable content to read more instruc
 Run the following command in a terminal:
 
 ```bash
-npx create-strapi-app@beta my-project --quickstart
+npx create-strapi@beta my-project --quickstart
```

Comment:
For v5 we should definitely remove the --quickstart option from the guide so users can see all the options instead. We consider it to be deprecated and will be removed in v6

Reply:
Yes, Alex! 
But if I do this, I need to update the rest of the Quick Start Guide (more questions to answer, etc.). 

I won't do it now, because the Quick Start Guide will again change in the following weeks (re. Cloud CLI) and it takes time to update it.

At least now they could use the shorter commands and (hopefully) have roughly the same experience than in the current Quick Start Guide ( = before the Cloud CLI)

Reply:
In short, for the final (stable, in September) version of the v5 Quick Start Guide I'll remove the --quickstart and ensure users experience the new workflow exactly like it's described. For now, too time consuming in regards of all the other tasks I have to do.

Reply:
:+1